apps/service-portal/src/components/Header/Header.tsx

```diff
@@ -1,31 +1,56 @@
-import React, { FC } from 'react'
+import React, { FC, useState } from 'react'
 import {
   Box,
   Hidden,
   Button,
   Logo,
   FocusableBox,
+  Icon,
 } from '@island.is/island-ui/core'
 import * as styles from './Header.css'
 import { ServicePortalPath } from '@island.is/service-portal/core'
+import { UserLanguageSwitcher } from '@island.is/shared/components'
 import { useLocale } from '@island.is/localization'
 import { UserMenu } from '@island.is/shared/components'
 import { useStore } from '../../store/stateProvider'
 import { ActionType } from '../../store/actions'
 import { BetaTag } from '../Logo/BetaTag'
```

Comment:
Can't we just remove the BetaTag component now that it is being removed? 😃

Reply:
whops - yes def 💯